pyfftw/interfaces/__init__.py

```diff
@@ -236,16 +236,16 @@
     del scipy
     from . import scipy_fftpack
 
-from distutils.version import LooseVersion
+from distutils.version import StrictVersion
 import numpy
 
 fft_wrap = None
-if LooseVersion(numpy.__version__) > LooseVersion("1.10"):
+if StrictVersion(numpy.__version__) >= StrictVersion("1.10.0"):
```

Comment:
I was going to suggest using
```python
if np.lib.NumpyVersion(np.__version__) >= '1.10.0':
```

to avoid the distutils import, but apparently that function was only added in numpy 1.9.  I think what you have is preferable in order to not break the import on older numpy.

Reply:
Interesting, did not know about that.